rust_src/src/crypto/mod.rs

```diff
@@ -418,7 +416,7 @@ fn _secure_hash(
         digest_size as EmacsInt
     };
     let digest = LispObject::from(unsafe { make_uninit_string(buffer_size as EmacsInt) });
-    let digest_str = digest.as_string_or_error();
+    let mut digest_str = digest.as_string_or_error();
```

Comment:
This is interesting: if it wasn't mutable before, why make it mutable now?

Reply:
Look a few lines down `digest_str.as_mut_slice()`